_docs/_latest/setup/migrate.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Migrate from Python Installer to Terraform Module
+order: 007
+---
+
+# {{ page.title }}
+
+This guide explains how to migrate a Forseti deployment from the
+deprecated Python Installer to the new Terraform module.
+
+---
+
+## Before You Begin
+
+Before you begin the migration process, you will need:
+
+- A version of the
+  [Terraform command-line interface](https://www.terraform.io/downloads.html)
+  in the 0.12 series
+- The domain and ID of the Google Cloud Platform (GCP) organization in
+  which Forseti is deployed.
+- The ID of the GCP project in which Forseti is deployed.
+- The suffix appended to the names of the Forseti resources; this is
+  likely a 7 digit number.
+- A
+  [key file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys)
+  for a service account in the organization with the
+  [roles required by the Terraform module](https://registry.terraform.io/modules/terraform-google-modules/forseti/google/4.1.0#iam-roles).
+
+## Configuring Terraform
+
+In the working directory, create a file named `main.tf` with content
```

Comment:
Add a paragraph above this for how to setup the key file, similar to the below.

---

Set an environment variable to the path of the service account JSON key file. 
This will be used by Terraform to authenticate with the GCP API.
```
export GOOGLE_APPLICATION_CREDENTIALS="PATH_TO_JSON_KEY_FILE"
```

Reply:
Fixed in 03247d5e5bafeabc6ced8547758247ee9901f54c.